packages/playwright/src/reporters/base.ts

```diff
@@ -420,10 +420,16 @@ export function formatResultFailure(screen: Screen, test: TestCase, result: Test
     });
   }
 
-  for (const error of result.errors) {
+  for (const [index, error] of result.errors.entries()) {
     const formattedError = formatError(screen, error);
+    let message = formattedError.message;
+    if (includePrompt) {
+      const promptPath = result.attachments.find(a => a.name === `_prompt-${index}`)?.path;
+      if (promptPath)
+        message += '\n' + screen.colors.cyan(`Error Prompt: ${relativeFilePath(screen, config, promptPath)}`);
```

Comment:
I think this is a layering violation. Test runner should not know about prompts.

Reply:
If the HTML reporter knows about prompts, why can't the terminal reporter?